src/components/SignatureRequest.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+import { useWallet } from '../providers/Wallet'
+import { getProviderFromUseWalletId } from '../ethereum-providers'
+import { signMessage } from '../lib/web3-utils'
+import { dayjs } from '../utils/date-utils'
+
+import signRequestSuccessIllustration from '../../src/assets/signRequestSuccess.svg'
+import signRequestFailIllustration from '../../src/assets/signRequestFail.svg'
+import signProcessing from '../../src/assets/loader.gif'
```

Comment:
Total nitpick, but let's use a better file name!

Reply:
signLoadingRing.gif? processingSignRing.gif?

Reply:
What do you think about `signRequestLoading.gif`, so it's similar to the other illustration names?